pkg/capicluster/cluster.go

```diff
@@ -0,0 +1,28 @@
+package capicluster
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/logging-operator/pkg/key"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Object struct {
+	client.Object
+}
+
+func (o Object) GetLoggingLabel() string {
+	labels := o.Object.GetLabels()
+
+	value := labels[key.LoggingLabel]
+
+	return value
```

Comment:
Maybe this function should return a boolean instead and not the whole label?

Reply:
You know what? It was actually hard to write it returning a string, because it would make so much more sense for it to return a boolean :rofl: 
But we were so focused on "return the contents of the label" that we persisted in returning a string.

WDYT @TheoBrigitte , do you see some reasons not to return a boolean?

Reply:
We discussed we keep it as is for the moment.

Reply:
We talked, let's keep it like this for now.